src/sliding_sync.rs

```diff
@@ -565,7 +582,61 @@ async fn async_worker(
                     }
                 });
             }
+            
+            MatrixRequest::GetFullyReadEvent { room_id } => {
+                let Some(room) = CLIENT.get().and_then(|c| c.get_room(&room_id)) else {
+                    error!("BUG: client/room not found for Get Fully Read Event {room_id}");
+                    continue;
+                };
+                
+                let _get_fully_read_event_task = Handle::current().spawn(async move {
+                    async fn fetch_fully_read_event(room: &Room, room_id: &OwnedRoomId) -> 
+                        Result<(OwnedEventId, MilliSecondsSinceUnixEpoch),  anyhow::Error> {
```

Comment:
proper formatting for a multi-line function signature is this:
```suggestion
                    async fn fetch_fully_read_event(
                        room: &Room,
                        room_id: &OwnedRoomId,
                    ) -> Result<(OwnedEventId, MilliSecondsSinceUnixEpoch),  anyhow::Error> {
```